python/core/auto_generated/vector/qgsvectorlayer.sip.in

```diff
@@ -1384,6 +1384,37 @@ Adds a ring to polygon/multipolygon features (takes ownership)
    changes can be discarded by calling :py:func:`~QgsVectorLayer.rollBack`.
 %End
 
+    Qgis::GeometryOperationResult addMultiRing( QgsCurve *ring /Transfer/, QgsFeatureIds *featureIds = 0 ) /PyName=addMultiCurvedRing/;
```

Comment:
Can we leave this in QgsVectorLayerEditUtils ONLY? These methods don't belong in the base QgsVectorLayer class in my opinion, and clutter up that API a lot.

Reply:
Thank you!

> If you're ok, I'll propose a cleanup for the other methods after the bugfix period.

Sounds great, it would be fantastic to get this cleaned up!